package.json

```diff
@@ -32,11 +32,11 @@
     "sinon": "^1.4.0 || ^2.1.0 || ^3.0.0 || ^4.0.0"
   },
   "devDependencies": {
-    "chai": "^4.1.0",
-    "eslint": "^3.19.0",
+    "chai": "^4.1.2",
+    "eslint": "^4.18.2",
```

Comment:
Let's leave dev dependency updates for a separate PR, and only update Sinon in this PR.